rclpy/rclpy/node.py

```diff
@@ -364,7 +364,6 @@ def declare_parameters(
         parameters: List[Union[
             Tuple[str],
             Tuple[str, Parameter.Type],
-            Tuple[str, Any],
```

Comment:
wasn't this for passing a value, e.g.: `declare_parameters('', [('my_param', 5)])`?

Reply:
I think the annotation was incorrect before. IIUC, we only expect a tuple up to length `3`. The last value of the tuple can have type `ParameterDescriptor` or `Any` (for the case of passing a value).

Reply:
https://github.com/ros2/rclpy/blob/1b335d37e52f401d207fcfe6ec59fce2290ee524/rclpy/test/test_node.py#L630

We actually can be more specific than using `Any`, I think `Union[int, float, str, bytes, None]` would be fine